pages/incinerator.tsx

```diff
@@ -11,6 +15,9 @@ import {
 import IncineratorInitManager from '@/views/incinerator/incinerator-init-manager';
 
 const IncineratorPage: NextPage = () => {
+  const { currentWallet } = useCurrentWallet();
```

Comment:
remove all change in this file, that we don't need because we have it already on InitManager